guides/common/modules/proc_enabling-capsule-in-UI.adoc

```diff
@@ -0,0 +1,25 @@
+[id="assigning-organization-location-capsule-server_{context}"]
+
+= Enabling the {SmartProxyServer} in the {Project} web UI
```

Comment:
What do you mean by Enabling? 
In this procedure, we assign {SmartProxy} to a correct organization and location. Please change the title.

Reply:
Changed